diff --git a/CMakeLists.txt b/CMakeLists.txt
index 568d73d7..0623b51a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -110,6 +110,7 @@ execute_process(COMMAND ${LSB_RELEASE_EXEC} -rs
 
 # Poppler
 option (BUILD_POPPLER "Build Poppler from git or other source" DEFAULT_BUILD_POPPLER)
+set (BUILD_POPPLER ON)
 
 ## Ubuntu 16.04 need a poppler build
 if (LSB_RELEASE_ID_SHORT STREQUAL "Ubuntu" AND LSB_RELEASE_NUMBER_SHORT STREQUAL "16.04")
